package.json

```diff
@@ -1,22 +1,31 @@
 {
   "name": "borsh",
-  "version": "0.7.0",
+  "version": "1.0.1",
   "description": "Binary Object Representation Serializer for Hashing",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "./lib/cjs/index.js",
+  "module": "./lib/esm/index.js",
+  "types": "./lib/types/index.d.ts",
   "files": [
     "lib",
     "LICENSE-APACHE",
     "LICENSE-MIT.txt"
   ],
   "scripts": {
-    "build": "tsc -p ./tsconfig.json",
     "test": "jest test --runInBand",
-    "fuzz": "jsfuzz borsh-ts/test/fuzz/borsh-roundtrip.js borsh-ts/test/fuzz/corpus/",
-    "dev": "yarn build -w",
     "pretest": "yarn build",
     "lint": "eslint borsh-ts/**/*.ts",
-    "fix": "eslint borsh-ts/**/*.ts --fix"
+    "fix": "eslint borsh-ts/**/*.ts --fix",
+    "compile": "tsc -b ./tsconfig.cjs.json ./tsconfig.esm.json ./tsconfig.types.json",
+    "build:clean": "rm -rf ./lib",
+    "build": "npm run build:clean && npm run compile && node .build_scripts/prepare-package-json.js"
+  },
+  "exports": {
+    ".": {
+      "types": "./lib/types/lib.d.ts",
+      "require": "./lib/cjs/lib.js",
+      "import": "./lib/esm/lib.js",
+      "default": "./lib/esm/lib.js"
```

Comment:
We had problem in importing. This seems should be index.d.ts/.js instead of lib.d.ts/.js